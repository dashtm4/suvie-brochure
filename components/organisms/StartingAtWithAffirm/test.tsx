import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { StartingAtWithAffirm } from '.'

describe('<StartingAtWithAffirm />', () => {
  it('should renders correct <StartingAtWithAffirm />', () => {
    const wrapper = shallow(<StartingAtWithAffirm />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })

  it('should has correct className from className prop', () => {
    const className = 'test-class-name'

    const wrapper = shallow(<StartingAtWithAffirm className={className} />)

    expect(wrapper.prop('className')).toContain(className)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
