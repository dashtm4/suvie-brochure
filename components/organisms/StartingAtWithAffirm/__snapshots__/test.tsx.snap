// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StartingAtWithAffirm /> should has correct className from className prop 1`] = `
<div
  className="test-class-name"
>
  * Starting at 0% APR with Affirm. 
  <PrequalifyNowLink />
</div>
`;

exports[`<StartingAtWithAffirm /> should renders correct <StartingAtWithAffirm /> 1`] = `
<div
  className=""
>
  * Starting at 0% APR with Affirm. 
  <PrequalifyNowLink />
</div>
`;
